Maths/armstrongNumber/armstrongNumber.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.8.0;
+
+/** @title This is a Solidity implementation to Armstrong number.
+  * @notice An Armstrong number is number that is equal to the sum of
+  * cubes of its digits.
+  * @author Wenceslas Sanchez (wenceslas-sanchez)
+  */
+contract ArmstrongNumber {
+    /** @notice Return the digit numbers of an integer.
+      * @dev From https://ethereum.stackexchange.com/questions/6623/can-i-check-the-number-of-digits-in-solidity
+      */
+    function numDigits(int _n) public pure returns (uint) {
+        uint8 digits = 0;
+        while (_n != 0) {
+            _n /= 10;
+            digits++;
+        }
+
+        return digits;
+    }
+
+    /** @notice Function to check whether a number is armstrong or not.
+      * @return `true` if the number is armstrong
+      */
+    function isArmstrongNumber(uint _n) public pure returns (bool) {
+        uint sum= 0;
```

Comment:
Thanks for the sol(idity) code!😊😊 I'm just curious if there is any standard formatter and linter in Solidity, so that we can set it up somehow.

Reply:
You are right, I have strange habits 😅 😄, we need a black like formatter => there is [Prettier](https://github.com/prettier-solidity/prettier-plugin-solidity)

Reply:
I know [Solhint ](https://github.com/protofire/solhint) too !

Reply:
PR #4

(Just to let you know @mkubdev, I choose Prettier because the repo is more active)

Reply:
I choose Prettier too. Much better than solhint after testing.

Reply:
#4